README.md

```diff
@@ -1,37 +1,96 @@
-# Job API (for data modeling)
+# Job API (for data modeling) &middot; ![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg) ![CI)](https://github.com/equinor/dm-job/actions/workflows/on-main-push.yaml/badge.svg)
+
+REST API for scheduling and running jobs in the Data Modelling Framework (DMF)
+
+## Usage
+
+This project delivers a container image, that is meant to be included in any deployment of DMF that need jobs.
```

Comment:
```suggestion
This project delivers a container image, that is meant to be included in any deployment of DMF that need jobs.
```
DMF?
I guess this should be DF (Development Framework), DM (Data Modelling), or something similar? At the very least, `DMF` should be explained/defined